lab_3_keywords_textrank/main.py

```diff
@@ -136,7 +153,10 @@ def encode(self, tokens: tuple[str, ...]) -> Optional[tuple[int, ...]]:
                 sequence of integer tokens
         In case of empty tokens input data, None is returned
         """
-        pass
+        if not tokens:
+            return None
+        self._learn_indices(tokens)
+        return tuple(self._word2id.values())
```

Comment:
Are you sure that it you need just values?

Reply:
done